galaxy_ng/app/api/v3/viewsets/sync.py

```diff
@@ -0,0 +1,22 @@
+from django.shortcuts import get_object_or_404
+from rest_framework import mixins
+from galaxy_ng.app.api import base as api_base
+from galaxy_ng.app.access_control import access_policy
+from pulp_ansible.app.models import CollectionRemote
+
+from ..serializers.sync import SyncConfigSerializer
+
+
+class SyncConfigViewSet(
+    mixins.RetrieveModelMixin,
+    mixins.UpdateModelMixin,
+    api_base.GenericViewSet,
+):
+    serializer_class = SyncConfigSerializer
+    permission_classes = [access_policy.CollectionRemoteAccessPolicy]
+
+    def get_object(self):
+        return get_object_or_404(
+            CollectionRemote,
+            basedistribution__name=self.kwargs['path']
```

Comment:
@newswangerd I am not 100% sure about this query, what do you think? Is it possible to have multiple CollectionRemote pointing to same basedistribution? if so it will raise a `Multiple Results Error`

Reply:
there's a one to many relation ship between distributions and repositories and another one to many relationship between repos and remotes, so that means that for every distrbution there should be no more than one repository associated with it and no more than one remote associated with it, so this should work